doc/filters.texi

```diff
@@ -10310,6 +10310,11 @@ The x and y offset coordinates where the box is drawn.
 @item h
 The width and height of the drawn box.
 
+@item box_source
+The box source of drawn box, which can be side_data_detection_bbox. So it will use box infomation in
+detecton bbox of side data, which includes @var{x}, @var{y} offset and @var{width}, @var{height}
+of drawn box. If box_source is set, the original x, y, width, height will be ignored.
```

Comment:
why 'original'?

Reply:
it means the value set manually by command (if they were set by user).
would it be better just delete 'original'?

Reply:
i think so